File: java/doit_java/Chapter7/src/arraylist/StudentTest.java
// 학생 성적 출력하기
package arraylist;

public class StudentTest {
    public static void main(String[] args) {
        
        Student studentLim = new Student(1001, "임채영");
        studentLim.addSubject("국어", 100);
        studentLim.addSubject("수학", 50);

        Student studentJung = new Student(1002, "정흥규");
        studentJung.addSubject("국어", 70);
        studentJung.addSubject("수학", 85);
        studentJung.addSubject("영어", 100);

        studentLim.showStudentInfo();
        System.out.println("\n========================================");
        studentJung.showStudentInfo();
    }
}

File: python/python_oop_practice/p_oop_2/chap06/array/student_test.py
from student import Student

def main():
    student_lee = Student(1001, "Lee")
    student_lee.add_subject("국어", 99)
    student_lee.add_subject("수학", 82)

    student_kim = Student(1002, "Kim")
    student_kim.add_subject("국어", 86)
    student_kim.add_subject("수학", 93)
    student_kim.add_subject("영어", 71)

    student_lee.show_student_info()
    print("==========================================")
    student_kim.show_student_info()


if __name__ == '__main__':
    main()
